sdk/typescript/src/cryptography/keypair.ts

```diff
@@ -83,11 +86,31 @@ export abstract class BaseSigner {
 	 * The public key for this keypair
 	 */
 	abstract getPublicKey(): PublicKey;
+
+	/**
+	 * The plain bytes for the secret key for this keypair.
+	 */
+	abstract getSecretKeyBytes(): Uint8Array;
```

Comment:
I think this needs to be on KeyPair rather than Signer.  We wanted to be able to create something like a wallet signer, or multisig signer which may not have secretKeyBytes

Reply:
Can we also make a `getSecretKey` that returns a string? I don't love that you need to use the `export` method, which really only exists for the wallet and should probably be deprecated anyway.